Makefile

```diff
@@ -50,6 +50,11 @@ install-dev:
 install-build:
 	pip install -r requirements/build.txt
 
+## install-ingest-s3:       install requirements for the s3 connector
+.PHONY: install-ingest-s3
```

Comment:
Should the the s3 ingest requirements be optional dependencies in `setup.py`?

Reply:
also added to setup.py (so following the local-inference pattern here)

Reply:
🎉